Documentation/connectors/ldap.md

```diff
@@ -43,6 +43,8 @@ Dex currently allows insecure connections because the project is still verifying
 
 User entries are expected to have an email attribute (configurable through `emailAttr`), and a display name attribute (configurable through `nameAttr`). `*Attr` attributes could be set to "DN" in situations where it is needed but not available elsewhere, and if "DN" attribute does not exist in the record.
 
+For the purposes of configuring this connector, "DN" is case-sensitive and should always be capitalised. This is treated as a special case because, while all other configuration values refer to single attributes on an entity, an LDAP Distinguished Name consists of multiple attributes which uniquely identify a resource.
```

Comment:
🤔 Does a DN really _consists of multiple attributes_? I've always though about these like file paths, referencing a node in directory tree... 
Looking at [`ldapwiki.com`](https://ldapwiki.com/wiki/Distinguished%20Names), I guess both is somewhat correct,
> A Distinguished Names is comprised of zero or more Relative Distinguished Name components that identify the location of the entry in the DIT. 

I guess this is OK, although I'd lean towards ditching the second sentence. What do you think? 😃

Reply:
I hesitated over the second sentence as well, let's just skip it

Reply:
(Updated)

Reply:
👍 thank you! 🎉